app/controllers/books_controller.rb

```diff
@@ -0,0 +1,52 @@
+class BooksController < ApplicationController
```

Comment:
:warning: Warning: [this card](https://trello.com/c/aJA6qzQr) only ask to create a Book model.
To create BooksController, we have [this card](https://trello.com/c/cquuRpQI) and many others

Reply:
:disappointed: we usually don't use rails generators :joy: